src/game/stats.h

```diff
@@ -11,4 +12,6 @@ int32_t Stats_GetPickups(void);
 int32_t Stats_GetKillables(void);
 int32_t Stats_GetSecrets(void);
 void Stats_Show(int32_t level_num);
-void Stats_ShowTotal(const char *filename);
+void Stats_ShowTotal(const char *filename, GAMEFLOW_LEVEL_TYPE level_type);
+void Stats_ComputeTotal(GAMEFLOW_LEVEL_TYPE level_type);
+bool Stats_CheckAllSecretsCollected();
```

Comment:
Let's add `(void)` to make it not possible to call the function with arbitrary arguments.